Bottom: 457ab1dddf0780cfc9d687f60a8b3cc2b9de11a7
Top:    bf23d4e44fce28ba64702edd40cd89e99e59a5a6
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-30 10:25:05 +0000

Refresh of add-foundcolor-to-resource-set

---

diff --git a/src/main.c b/src/main.c
index d80612d..e41fc55 100644
--- a/src/main.c
+++ b/src/main.c
@@ -770,11 +770,11 @@ Color to indicate physical connections. Default: @samp{#00ff00}
 /* %start-doc options "3 Colors"
 @ftable @code
 @item --found-color <string>
-Color to indicate logical connections. Default: @samp{#ff00ff}
+Color to indicate logical connections. Default: @samp{#00ff00}
 @end ftable
 %end-doc
 */
-  COLOR (FoundColor, "#ff00ff", "found-color",
+  COLOR (FoundColor, "#00ff00", "found-color",
 	 "color to indicate logically connected objects"),
 
 /* %start-doc options "3 Colors"
